components/script/dom/worker.rs

```diff
@@ -188,20 +188,20 @@ impl Runnable for WorkerMessageHandler {
     }
 }
 
-pub struct WorkerEventHandler {
+pub struct SimpleWorkerErrorHandler {
```

Comment:
I had to figure out what it does in order to categorize the runnable. Since it handles errors I felt "WorkerEventHandler" to be *too* generic. Of course I'll revert it, if you want…